pkg/controllers/resourcechange/resourcechange_controller.go

```diff
@@ -215,47 +265,74 @@ func (r *Reconciler) getUnstructuredObject(objectKey keys.ClusterWideKey) (runti
 	return obj, isClusterScoped, nil
 }
 
-// triggerAffectedPlacementsForUpdatedClusterRes find the affected placements for a given updated cluster scoped resources.
-func (r *Reconciler) triggerAffectedPlacementsForUpdatedClusterRes(key keys.ClusterWideKey, res *unstructured.Unstructured) (ctrl.Result, error) {
-	if r.PlacementControllerV1Alpha1 != nil {
-		// List all the CRPs.
-		crpList, err := r.InformerManager.Lister(utils.ClusterResourcePlacementV1Alpha1GVR).List(labels.Everything())
-		if err != nil {
-			return ctrl.Result{}, fmt.Errorf("failed to list all the v1alpha1 cluster placements: %w", err)
-		}
+// triggerAffectedPlacementsForUpdatedClusterRes find the affected placements for a given updated cluster scoped or namespace scoped resources.
+// If triggerCRP is true, it will trigger the cluster resource placement controller, otherwise it will trigger the resource placement controller.
+func (r *Reconciler) triggerAffectedPlacementsForUpdatedClusterRes(key keys.ClusterWideKey, res *unstructured.Unstructured, triggerCRP bool) (ctrl.Result, error) {
```

Comment:
The function name needs a change: `triggerAffectedPlacementsForUpdatedRes`.

Reply:
i will refactor the func name in my next PR to reduce the confusion.